tools/cmock/ota.yml

```diff
@@ -0,0 +1,45 @@
+:cmock:
+  :mock_prefix: mock_
+  :when_no_prototypes: :warn
+  :enforce_strict_ordering: true
+  # fclose is called after program terminates, so this will avoid a segfault.
+  :fail_on_unexpected_calls: false
```

Comment:
Would it be possible to make `fclose` an alias as well? The disadvantage with this option is that unexpected calls to mocked functions will not cause a test failure.

Reply:
It should be possible, I'll try this.

Reply:
Okay, this is minor, so I'll approve this PR now.

Reply:
Thanks, I tried just now and there is still a seg fault, even after creating a mock. I'll have to look more into this to see what is causing this.

Reply:
No worries. I had problems getting around this too. I was just curious if your new approach would fix it.